test-yolo.py

```diff
@@ -0,0 +1,262 @@
+        # retreive local copy of the dev repo:
+        command = shlex.split("terminus site:info %s --field=id" % siteName)
+        siteID = subprocess.check_output(command)
+        command = shlex.split("terminus connection:set %s.dev git" % siteName)
+        subprocess.call(command)
+        command = ['git', 'clone', 'ssh://codeserver.dev.%s@codeserver.dev.%s.drush.in:2222/~/repository.git' % (siteID[:-1], siteID[:-1]), '%s' % siteName]
+        subprocess.call(command)
+        curr = os.getcwd()
+        os.chdir(curr + '/%s' % siteName)
+
+        #set up the site environment:
+        if siteName == "ci-drupal-7":
+            version = "7.x-2.12"
+        else:
+            version = "8.x-1.7"
+
+        command = shlex.split("terminus connection:set %s.dev sftp" % siteName)
+        subprocess.call(command)
+        command = shlex.split("terminus --yes drush -- %s.dev en easy_breadcrumb -y" % siteName)
+        subprocess.call(command)
+        command = shlex.split("terminus --yes drush -- %s.dev dl easy_breadcrumb-%s -y" % (siteName, version))
+        subprocess.call(command)
+
+        # update the code, but not the database yet, and deploy to test
+        command = shlex.split("terminus --yes drush -- %s.dev pm-updatecode -y" % siteName)
+        subprocess.check_output(command)
+        command = shlex.split("terminus env:commit %s.dev --message 'updated code'" % siteName)
+        subprocess.check_output(command)
+        command = shlex.split("terminus env:deploy %s.test" % siteName)
+        subprocess.call(command)
+
+        # check that database updates are needed
+        command = shlex.split("terminus drush -- %s.test updatedb -n" % siteName)
+        test1, err1 = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+        print(test1)
+        assert "updates" in test1
+        assert "pending" in test1
+        
+        # update the database on dev, clone to test
+        command = shlex.split("terminus --yes drush -- %s.dev updatedb -y" % siteName)
+        subprocess.call(command)
+        command = shlex.split("terminus env:clone %s.dev test" % siteName)
+        subprocess.call(command)
+
+        # test that deploy caused updates
+        command = shlex.split("terminus drush %s.test updatedb" % siteName)
+        err2 = subprocess.check_output(command, stderr=subprocess.STDOUT)
+        assert "No database updates required" in err2
+
+        #reset to original state
+        self.resetRoutine()
+
+
+class CacheClearTestCase(unittest.TestCase):
+    def makeMultidev(self):
+        command = shlex.split("terminus multidev:create %s.dev temp" % siteName)
+        subprocess.call(command)
+        command = shlex.split("terminus --yes multidev:merge-from-dev --updatedb %s.temp" % siteName)
+        subprocess.check_output(command)
+
+    #note: I think env:deploy --cc doesn't actually clear the cache (looking at workflows doesn't show a cache clearing workflow happening)
```

Comment:
If you leave the test uncommented, then you can add `@unittest.skip("Known bug: env::deploy does not clear the cache")`